### ISSUE TITLE
What is alpha_ts, beta_ts, Cdt?

### ISSUE THREAD

**Issue by Richarizardd** [wlyra/covid19#3]
In your RK3 function, was wondering how you calculated: alpha_ts, beta_ts, and Cdt

**Comment by jcert** [comment]
It is some implementation of a third-order [Runge-Kutta method](https://en.wikipedia.org/wiki/List_of_Runge%E2%80%93Kutta_methods#Kutta's_third-order_method). But I can't really figure which third method it is and how those coefficients were calculated.

**Comment by wlyra** [comment]
Yes, these are the coefficients for a low storage 3rd order Runge-Kutta,
from Williamson et al. 1980

https://www.sciencedirect.com/science/article/pii/0021999180900339?via%3Dihub

Also explained here

https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/19940029698.pdf

and here (appendix H4)

http://pencil-code.nordita.org/doc/manual.pdf
